layers/meta-balena-beaglebone/recipes-bsp/u-boot/u-boot-bb.org_%.bbappend

```diff
@@ -2,23 +2,61 @@ FILESEXTRAPATHS:append := ":${THISDIR}/${PN}"
 
 inherit resin-u-boot
 
-SRC_URI += " \
+SRC_URI:append:beaglebone-ai64 = " \
```

Comment:
Wouldn't this break all other beaglebone devices like beaglebone, beaglebone green, etc?